guides/schematics.md

```diff
@@ -13,48 +13,92 @@ install schematic.
 ng add @angular/material
 ```
 
-The install schematic will help you add Material to a new project. 
-This schematic will:
+In case you just want to install the `@angular/cdk`, there are also schematics for the [Component Dev Kit](https://material.angular.io/cdk)
+
+```
+ng add @angular/cdk
+```
+
+The Angular Material `ng add` schematic helps you setup an Angular CLI project that uses Material. Running `ng add` will:
 
 - Ensure [project dependencies](./getting-started#step-1-install-angular-material-angular-cdk-and-angular-animations) are placed in `package.json`
 - Enable the [BrowserAnimationsModule](./getting-started#step-2-configure-animations) your app module
 - Add either a [prebuilt theme](./theming#using-a-pre-built-theme) or a [custom theme](./theming#defining-a-custom-theme)
-- Add Roboto fonts to your index.html
-- Add the [Material Icon font](./getting-started#step-6-optional-add-material-icons) to your index.html
+- Add Roboto fonts to your `index.html`
+- Add the [Material Icon font](./getting-started#step-6-optional-add-material-icons) to your `index.html`
 - Add global styles to
   - Remove margins from `body`
   - Set `height: 100%` on `html` and `body`
   - Make Roboto the default font of your app
-- Install and import `hammerjs` for [touch gesture support](./getting-started#step-5-gesture-support) in your project
+- Install and import `hammerjs` for [gesture support](./getting-started#step-5-gesture-support) in your project
+
+
+## Component schematics
+In addition to the install schematic, Angular Material comes with multiple schematics that can be
+used to easily generate Material Design components:
+
+- [Address form](#address-form-schematic)
+- [Dashboard](#dashboard-schematic)
+- [Navigation](#navigation-schematic)
+- [Table](#table-schematic)
+- [Tree](#tree-schematic)
```

Comment:
Updated. I tried my best explaining the schematics in a short but accurate way. Also, yeah I agree that it would make sense to improve that whole document at some point.